### PR TITLE
JF/feature/#47 orders setup

### LINKED ISSUE #47: Setup oders - some fixes: Controller, Service, Repo, Dao

### PR DESCRIPTION
**Pull request by JaafarF**

### CODE REVIEW

File: online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/user/User.java [hunk 1]
```diff
@@ -58,16 +61,30 @@ public class User {
...
         this.password = password;
         this.phoneNumber = phoneNumber;
         this.role = role;
         this.addresses = addresses;
+        this.orders = orders;
+    }
+
+    public void add(Order order) {
+        if (order != null) {
+            if (orders == null) {
```
Comment (sarajaned) [inline online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/user/User.java]: Orders is initialised above when declared on line 65

File: online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/OnlineBookStoreApplication.java
Comment (sarajaned) [inline online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/OnlineBookStoreApplication.java:1]: unused imports

File: online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/address/AddressJpaService.java [hunk 2]
```diff
@@ -32,4 +32,9 @@ public boolean existAddressById(long id) {
     public Optional<Address> getAddressById(Long addressId) {
         return addressRepository.findById(addressId);
     }
+
+    @Override
+    public Address getAddressReferenceById(Long addressId) {
```
Comment (sarajaned) [inline online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/address/AddressJpaService.java]: Is this used or will it be used at a later point?

**Review by sarajaned — CHANGES_REQUESTED** [review]
Just a couple of small things I noticed

File: online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/user/User.java (hunk 1 above)
Comment (JaafarF) [reply online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/user/User.java]: You are right, 
and I just even figured out that this method is never used, so I removed it.

File: online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/OnlineBookStoreApplication.java
Comment (JaafarF) [reply online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/OnlineBookStoreApplication.java:1]: I removed unused imports from other files also.

File: online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/address/AddressJpaService.java (hunk 2 above)
Comment (JaafarF) [reply online-book-store/src/main/java/com/amigoscode/cohort2d/onlinebookstore/address/AddressJpaService.java]: Removed

**Review by sarajaned — APPROVED** [review]

**Pull request merged**